vm/src/stdlib/io.rs

```diff
@@ -3068,23 +3061,25 @@ mod _io {
         }
     }
 
-    #[pyimpl(flags(BASETYPE, HAS_DICT), with(PyRef))]
-    impl StringIO {
-        fn buffer(&self, vm: &VirtualMachine) -> PyResult<PyRwLockWriteGuard<'_, BufferedIO>> {
-            if !self.closed.load() {
-                Ok(self.buffer.write())
-            } else {
-                Err(io_closed_error(vm))
-            }
-        }
+    #[derive(FromArgs)]
+    struct StringIONewArgs {
+        #[pyarg(positional, optional)]
+        object: OptionalOption<PyStrRef>,
 
-        #[pyslot]
-        fn tp_new(
+        // TODO: use this
+        #[pyarg(any, default)]
+        #[allow(dead_code)]
+        _newline: Newlines,
```

Comment:
the argument name must be `newline` without `_` because tests expect the exact keyword argument name